Assets/Scripts/Interactions/State.cs

```diff
@@ -1,4 +1,22 @@
-﻿public abstract class State
+﻿using UnityEngine;
+public abstract class State
 {
-	public abstract void Update(Interaction inter);
+//	public abstract void Update (Interaction inter);
+	
+	public virtual void Update(Interaction inter)
+	{
+		Ray ray = Camera.main.ScreenPointToRay (Input.mousePosition);
+		RaycastHit hit;
+
+		if (Physics.Raycast (ray, out hit, 100))
+		{
+			Description desc = hit.collider.gameObject.GetComponent<Description> ();
+			if (!Util.IsNull (desc))
+			{
+				MarketHex hex = hit.collider.gameObject.GetComponent<MarketHex> ();
```

Comment:
You should store the data for the tooltip in the `Description` component and get that instead of the `MarketHex`.